test/integration_test/s3/getObject.spec.js

```diff
@@ -74,7 +80,11 @@ describe('download integrity', function() {
       Key: 'key',
     };
     s3 = new AWS.S3({responseChecksumAlgorithm: 'md5'});
-    putObject({Body: 'this is a test!'}, done);
+    createBucket({}, done);
```

Comment:
Can you pass the bucket name into this function? Right now I would have to look at the `createBucket` code to understand what bucket name is being chosen and how, and I don't know what the empty object is for.